docs/sync.js

```diff
@@ -876,6 +888,18 @@ class User {
    * @returns {Realm.User~Push}
    */
   push(serviceName) {}
+
+  /**
+   * Adds a listener that will be fired on various events, including token refresh
```

Comment:
Can we be more precise? And do we need to be more precise?

Reply:
I've added more precise events.